### PR TITLE
Create Clickbench for BYOB that run every hour instead of 30mins

### PR DESCRIPTION
**Pull request by a-khaledf**

### CODE REVIEW

File: .github/workflows/axiom-byob-clickbench.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+
+jobs:
+  clickbench:
+    runs-on: ubuntu-latest
+    env:
+      AXIOM_URL: ${{ secrets.AXIOM_URL_BYOB }}
```
Comment (tsenart) [inline .github/workflows/axiom-byob-clickbench.yml:18]: Can we only make secret what needs to be? i.e. TOKEN

Comment (tsenart) [reply .github/workflows/axiom-byob-clickbench.yml:18]: We should create this in the axiom repo I think.

Comment (a-khaledf) [reply .github/workflows/axiom-byob-clickbench.yml:18]: do you wanna move everything to axiom repo? i can do that

Comment (tsenart) [reply .github/workflows/axiom-byob-clickbench.yml:18]: Yeah I think that makes sense.

Comment (a-khaledf) [reply .github/workflows/axiom-byob-clickbench.yml:18]: I didn't want the main clickbench to run here and the byob to run in axiom repo for consistency. it is either all lives here or all moved there.

**Comment by a-khaledf** [comment]
Moving the worklfows to the main axiom repository

**Pull request closed without merging**